pkg/webui/account/containers/authorizations-table/index.js

```diff
@@ -76,7 +76,7 @@ const OAuthClientAuthorizationsTable = props => {
     [],
   )
 
-  const getItems = React.useCallback(() => getAuthorizationsList(userId, []), [userId])
+  const getItems = React.useCallback(filters => getAuthorizationsList(userId, ...filters), [userId])
```

Comment:
```suggestion
  const getItems = React.useCallback(filters => getAuthorizationsList(userId, filters), [userId])
```
You cannot spread an object into function arguments. Pagination actions expect the `filters`-object as second argument.